lib/matplotlib/tests/test_figure.py

```diff
@@ -381,6 +382,34 @@ def test_warn_cl_plus_tl():
     assert not(fig.get_constrained_layout())
 
 
+def test_add_artist():
```

Comment:
You may want to (but don't have to) use https://github.com/matplotlib/matplotlib/pull/11408 for the test, which has the benefit of directly taking care of all three output formats and generating failure diffs when needed.

Reply:
Ok, did that. Locally, the svg test always skips though. Not sure about the reason.

Reply:
You just don't have inkscape installed.  I wouldn't worry about it...

Reply:
I have several inkscape versions installed, maybe that's the problem.

Reply:
...sorry, shouldn't have assumed; rephrase - inkscape wasn't callable for some reason :wink: